plugins/fusion/qt.py

```diff
@@ -524,6 +530,20 @@ def run(self):
         return self.password
 
 
+class DisabledFusionButton(StatusBarButton):
+    def __init__(self, wallet, message):
+        super().__init__(icon_fusion_logo_gray, 'Fusion', self.show_message)
+
+        self.wallet = wallet
+        self.message = message
+
+        self.setToolTip(_("CashFusion ({disabled_message})").format(
+            disabled_message=message))
+
+    def show_message(self):
+        QMessageBox.critical(Plugin.get_suitable_dialog_window_parent(self.wallet),
```

Comment:
1. I'm glad you used that `get_suitable_dialoig_window_parent` here 👍 
2. I wonder if a less .. *alarming* `QMessageBox` is warranted here.  On macOS at least this dialog box looks a little bit scary (big red stop sign)... maybe `QMessageBox.information`?

Reply:
updated